.travis/linux_openresty_runner.sh

```diff
@@ -169,7 +169,8 @@ script() {
     sudo bash ./utils/check-plugins-code.sh
 
     make lint && make license-check || exit 1
-    APISIX_ENABLE_LUACOV=1 PERL5LIB=.:$PERL5LIB prove -Itest-nginx/lib -r t
+    # APISIX_ENABLE_LUACOV=1 PERL5LIB=.:$PERL5LIB prove -Itest-nginx/lib -r t
```

Comment:
Could we remove this comment line?

Reply:
we may enable it later, so I think we should keep the original version.